app/scripts/controllers/push-platform-notifications/push-platform-notifications.ts

```diff
@@ -113,7 +149,11 @@ export class PushPlatformNotificationsController extends BaseController<
 
     try {
       // 2. Call the activatePushNotifications method from PushPlatformNotificationsUtils
-      const regToken = await activatePushNotifications(bearerToken, UUIDs);
+      const regToken = await activatePushNotifications(
+        bearerToken,
+        UUIDs,
+        this.#metamaskNotification.updateMetamaskNotificationsList,
```

Comment:
Why do we do this?

Reply:
Every time I receive a push notification, I also update the list of notifications managed by the controller. This way, if the user clicks on the notification, it will be available to show the detail page